src/core/MeganavContentProducts/component.jsx

```diff
@@ -23,57 +23,36 @@ const MeganavContentPlatform = ({ paths, absUrl }) => (
+            <a href={absUrl("/solutions/channels")} className="group ui-meganav-media">
+              <p className="ui-meganav-media-heading">Pub/Sub Channels</p>
+              <p className="ui-meganav-media-copy">Build infinitely scalable realtime applications.</p>
             </a>
           </li>
           <li>
-            <a href={absUrl("/examples/emoji-reactions")} className="ui-meganav-media group">
-              <p className="ui-meganav-media-heading">Emoji Reactions</p>
-              <p className="ui-meganav-media-copy">React with an emoji to a message.</p>
+            <a href={absUrl("/spaces")} className="group ui-meganav-media">
+              <p className="ui-meganav-media-heading">Spaces (Beta)</p>
+              <p className="ui-meganav-media-copy">Create multi-user collaborative environments.</p>
             </a>
           </li>
           <li>
-            <a href={absUrl("/examples/activity-feed")} className="ui-meganav-media group">
-              <p className="ui-meganav-media-heading">Activity Feed</p>
-              <p className="ui-meganav-media-copy">Display a list of user actions in realtime.</p>
-            </a>
-          </li>
-          <li>
-            <a href={absUrl("/examples/live-cursors")} className="ui-meganav-media group">
-              <p className="ui-meganav-media-heading">Live Cursors</p>
-              <p className="ui-meganav-media-copy">Track all cursors in realtime.</p>
-            </a>
-          </li>
-          <li>
-            <a href={absUrl("/examples/typing-indicator")} className="ui-meganav-media group">
-              <p className="ui-meganav-media-heading">Typing Indicator</p>
-              <p className="ui-meganav-media-copy">See when a user is typing a message.</p>
-            </a>
-          </li>
-          <li>
-            <a href={absUrl("/examples/chat-admin-privileges")} className="ui-meganav-media group">
-              <p className="ui-meganav-media-heading">Chat Admin Privileges</p>
-              <p className="ui-meganav-media-copy">Control who can take admin actions in a digital space.</p>
+            <a href={absUrl("/livesync")} className="group ui-meganav-media">
+              <p className="ui-meganav-media-heading">LiveSync (Pre release)</p>
+              <p className="ui-meganav-media-copy">Keep clients in sync with any relational database.</p>
             </a>
           </li>
         </ul>
-        <FeaturedLink url={absUrl("/examples")} textSize="text-p3">
-          Explore all live examples
-        </FeaturedLink>
       </div>
 
       <div className="col-span-full md:col-span-4 pt-8 pb-24 md:py-24 lg:py-32 px-24 sm:px-32 md:px-0 bg-white">
-        <h3 className="ui-meganav-overline" id="meganav-platform-panel-list-our-technology">
+        <h3 className="ui-meganav-overline" id="meganav-products-panel-list-our-technology">
           Our technology
```

Comment:
@dpiatek we discovered that there is a difference between the copy here between the React and Ruby components, and that the screenshot/parity tests didn't pick it up until we were working on #283. We'll fix it in the other PR, but just something to be aware of